### ISSUE TITLE
Can't close menu

### ISSUE THREAD

**Issue by Nimplex** [ThymonA/menuv#25]
im writing simple menu using MenuV, and i have some issues when closing menu.

Code:
```lua
local button2 = menu:AddButton({ icon = '💀', label = 'Arenka 2' })
button2:On('select', function(item)
    teleportToCoords(GetPlayerPed(-1), 1934.6615, 4715.15771, 41.09946, 288.548676)
    menu:Close()
end)
```

I wanted menu to close after selecting button but instead it's still visible but buttons dont work anymore.

**Comment by ThymonA** [comment]
Issue solved, contacted with `Nimplex#1010` on Discord. Bug fixed [`c406741`](https://github.com/ThymonA/menuv/commit/f701aa40c06c6536c93930fb841435d3645b85fc)

**Comment by robertne13** [comment]
could you post the solution?
